usmqe_tests/api/ceph/test_pool.py

```diff
@@ -0,0 +1,587 @@
+    )
+    """@pylatest api/ceph.update_pool_name_valid
+        API-ceph: update_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        .. test_step:: 2
+
+            Check if changes are made in Ceph pool.
+
+        .. test_result:: 2
+
+           Updates are done in pool.
+            """
+
+    storage = ceph_cluster.CephCluster(pytest.config.getini("usm_ceph_cl_name"))
+    pools = storage.osd.pool_ls(detail=True)
+    LOGGER.info("List of Ceph pools:{}".format(pools))
+    selected_pools = [pool for pool in pools
+                      if pool["pool_name"] == valid_pool_name
+                      ]
+    pytest.check(len(selected_pools) == 1,
+                 "Pool {} should be updated in Ceph \
+                 cluster {}.".format(valid_pool_name,
+                                     pytest.config.getini("usm_ceph_cl_name")),
+                 issue="https://github.com/Tendrl/ceph-integration/issues/225"
+                 )
+
+
+def test_update_pool_valid(valid_cluster_id,
+                           valid_pool_id,
+                           valid_session_credentials,
+                           valid_pool_name,
+                           valid_size,
+                           valid_minsize,
+                           valid_pg_num):
+    # TODO add quota support
+    """@pylatest api/ceph.update_pool_valid
+        API-ceph: update_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        Description
+        ===========
+        Negative Update from CRUD for pools.
+
+        Update existing pool with valid data via API.
+
+        .. test_step:: 1
+
+            Connect to Tendrl API via GET request
+            to ``APIURL/:cluster_id/CephUpdatePool``
+            Where cluster_id is set to predefined value.
+
+        .. test_result:: 1
+
+           Update job should pass.
+            """
+    valid_pool_name = valid_pool_name + "_updated"
+    valid_size = valid_size + 1
+    valid_minsize = valid_minsize + 1
+    valid_pg_num = valid_pg_num * 2
+    api = cephapi.TendrlApiCeph(auth=valid_session_credentials)
+    job_id = api.update_pool(valid_cluster_id, valid_pool_id,
+                             size=valid_size, min_size=valid_minsize,
+                             pg_num=valid_pg_num)["job_id"]
+    api.wait_for_job_status(
+        job_id,
+        issue="https://github.com/Tendrl/ceph-integration/issues/225"
+        )
+    """@pylatest api/ceph.update_pool_valid
+        API-ceph: update_pool
+        ******************************
+
+        :authors:
+            - fbalak@redhat.com
+            - mkudlej@redhat.com
+
+        .. test_step:: 2
+
+            Check if changes are made in Ceph pool.
+
+        .. test_result:: 2
+
+           Updates are done in pool.
+            """
```

Comment:
Yeah, I proposed to not use them at all until we figure out https://gitlab.com/mbukatov/pylatest/issues/31 , but not everyone agreed and it doesn't break anything (but readability a bit).